samples/stock/demo/two-standalone-navigators/demo.js

```diff
@@ -0,0 +1,123 @@
+const commonOptions = {
+    api: {
+        url: 'https://demo-live-data.highcharts.com',
+        access: {
+            url: 'https://demo-live-data.highcharts.com/token/oauth',
+            username: 'username',
+            password: 'password'
+        }
+    }
+};
+
+const AMDISIN = 'US0079031078';
+
+// eslint-disable-next-line no-undef
+const AMDPriceConnector = new Connectors.Morningstar.TimeSeriesConnector({
+    ...commonOptions,
+    series: {
+        type: 'Price'
+    },
+    securities: [
+        {
+            id: AMDISIN,
+            idType: 'ISIN'
+        }
+    ],
+    currencyId: 'EUR'
+});
+
+Promise.all([AMDPriceConnector.load()]).then(() => {
+    const cols = AMDPriceConnector.table.getColumns();
+
+    const name = Array.from(Object.keys(cols).filter(k => k !== 'Date'))[0];
+    const price = cols[name].map((value, i) => [cols.Date[i], value]);
+
+    const firstNav = Highcharts.navigator('navigator-container', {
+        series: [{
+            data: price
+        }]
+    });
+
+    const secondNav = Highcharts.navigator('second-navigator-container', {
+        series: [{
+            data: price
+        }]
+    });
+
+    const priceChart = Highcharts.stockChart('price-chart', {
+        navigator: {
+            enabled: false
+        },
+        rangeSelector: {
+            enabled: false
+        },
+        scrollbar: {
+            enabled: false
+        },
+        title: {
+            text: 'AMD Stock Pice'
+        },
+        series: [{
+            name: 'AMD',
+            data: price
+        }]
+    });
+
+    // Adjust second navigator range based on the first navigator
+    Highcharts.addEvent(firstNav.navigator, 'setRange', function (e) {
+        secondNav.navigator.xAxis.update({ min: e.min, max: e.max });
+
+        const { min, max } = secondNav.navigator.chart.xAxis[0].getExtremes();
+        const oneYear = 24 * 3600 * 1000 * 365; // 1 year
+
+        let newMin = Math.max(min, e.min);
+        let newMax = Math.min(max, e.max);
+
+        // Ensure the minimal range
+        if (newMax - newMin < oneYear) {
+            if (newMax < max) {
+                newMin = newMax - oneYear;
+            } else {
+                newMax = newMin + oneYear;
+            }
+        }
+
+        e.min = newMin;
+        e.max = newMax;
+
+        Highcharts.fireEvent(secondNav.navigator, 'setRange', e);
+    });
+
+    // Add plot band on first navigator to indicate the range of the chart and
+    // second navigator
+    Highcharts.addEvent(secondNav.navigator, 'setRange', function (e) {
+        firstNav.navigator.xAxis.update({
+            plotBands: [{
+                from: e.min,
+                to: e.max,
+                color: '#9E66FF'
```

Comment:
Find better contrasting color, unless @marvin19 thinks its fine

Reply:
I agree on the contrasting colors. Series line to background should be 3:1